### PR TITLE
Style search results

### PR DESCRIPTION
**Pull request by stormcloud266**
Styles the search results pop up.

Screenshot 2024-05-08 at 1 25 00 PM

### CODE REVIEW

**Comment by stormcloud266** [comment]
* **#395** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/395?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `dev`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @stormcloud266 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: packages/app/features/activity/screen.tsx [hunk 1]
```diff
@@ -132,65 +130,170 @@ function ActivityBody() {
...
+      br="$5"
+      key={item.send_id}
+      width="100%"
+      $gtLg={{
+        width: 'calc((100% - 48px) / 3)',
+        bc: rowBC,
+        p: '$1.5',
+      }}
+      $gtXl={{
+        width: 'calc((100% - 72px) / 4)',
```
Comment (0xBigBoss) [inline packages/app/features/activity/screen.tsx:248]: sadly, we can't do this on native. not sure what the correct usage would be on native to match the style though.

```suggestion
        width: isWeb ? 'calc((100% - 48px) / 3)' : '100%',
```

File: packages/app/features/activity/screen.tsx [hunk 2]
```diff
@@ -132,65 +130,170 @@ function ActivityBody() {
...
+              {item.tag_name ? `@${item.tag_name}` : `#${item.send_id}`}
+            </Text>
+          </YStack>
+        </XStack>
+      </Link>
+    </View>
+  )
+}
+
 // TODO: Replace with dynamic list
```
Comment (0xBigBoss) [inline packages/app/features/activity/screen.tsx:297]: we're getting to the point where let's jsut hide shit that doesn't work. can u hide this component?

File: packages/app/features/activity/screen.tsx [hunk 3]
```diff
@@ -382,31 +485,59 @@ function Row({ activity }: { activity: (typeof activities)[number] }) {
...
+        <SchemaForm
+          form={form}
+          defaultValues={{ query: '' }}
+          onSubmit={() => {
+            // noop
+          }}
+          schema={SearchSchema}
+          props={{
+            query: {
+              placeholder: 'Name, $Sendtag, Phone, Email',
```
Comment (0xBigBoss) [inline packages/app/features/activity/screen.tsx:503]: i posted on Victor's PR too. Email doesn't apply.

File: packages/ui/src/themes/colors.ts [hunk 4]
```diff
@@ -267,4 +267,7 @@ export const brandColors = {
   decay: '#3E4A3C',
   darkest: '#081619',
   jet: '#343434',
+  metalTouch: '#1C2A2D',
+  darkGrayTextField: '#666666', // hsla(0, 0%, 40%, 1)
```
Comment (0xBigBoss) [inline packages/ui/src/themes/colors.ts:271]: 😈 🤘

Comment (0xBigBoss) [reply packages/ui/src/themes/colors.ts:271]: we missed a big op here to call this metal touch.

File: packages/app/features/activity/screen.tsx [hunk 5]
```diff
@@ -132,65 +130,170 @@ function ActivityBody() {
...
+        animation="quick"
+        gap="$size.2.5"
+        mt="$size.3.5"
+        width="100%"
+        enterStyle={{
+          opacity: 0,
+          y: -10,
+        }}
+      >
+        {Object.entries(results).map(([keyField, value]) =>
```
Comment (0xBigBoss) [inline packages/app/features/activity/screen.tsx:172]: does this ensure ordering correctly?

I think it shuold be phone, sendtag, then sendid? let's confirm with ethen and everyone in chat.

**Review by 0xBigBoss — APPROVED** [review]
just a few comments to address. lgtm, didn't see it in action though.

File: packages/ui/src/themes/colors.ts (hunk 4 above)
Comment (stormcloud266) [reply packages/ui/src/themes/colors.ts:271]: lol sure did damn. well `theDevilsTextField` is still available at least

File: packages/app/features/activity/screen.tsx (hunk 2 above)
Comment (stormcloud266) [reply packages/app/features/activity/screen.tsx:297]: It does display correctly, it's just that the info is hardcoded fake data. do you want me to take it out until we get the real data working?

File: packages/app/features/activity/screen.tsx (hunk 1 above)
Comment (stormcloud266) [reply packages/app/features/activity/screen.tsx:248]: luckily it's just the 100% on mobile, so i'll just add that ternary in there

File: packages/app/features/activity/screen.tsx (hunk 2 above)
Comment (0xBigBoss) [reply packages/app/features/activity/screen.tsx:293]: yeah, just hide it for now.

File: packages/app/features/activity/screen.tsx [hunk 6]
```diff
@@ -382,31 +481,59 @@ function Row({ activity }: { activity: (typeof activities)[number] }) {
...
+        <SchemaForm
+          form={form}
+          defaultValues={{ query: '' }}
+          onSubmit={() => {
+            // noop
+          }}
+          schema={SearchSchema}
+          props={{
+            query: {
+              placeholder: 'Name, $Sendtag, Phone',
```
Comment (0xBigBoss) [inline packages/app/features/activity/screen.tsx:499]: sorry, my bad, name doesn't work here either. It's just SENDTAG, SEND ID, PHONE

**Comment by stormcloud266** [comment]
### Merge activity

* **May 10, 11:15 PM EDT**: @stormcloud266 started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/0xsend/sendapp/395).

**Pull request merged**